rtsp.go

```diff
@@ -258,7 +259,7 @@ func (rc *rtspCamera) closeConnection() {
 		rc.client.Close()
 		rc.client = nil
 	}
-	rc.resetAU(nil)
+	rc.resetLazyAU(nil)
```

Comment:
Do we want this to be reste to nil? Okay to leave as is in case it opens up a can of worm.

Reply:
changed to an empty [][]byte{}